src/app/components/AccountInfo/NodeUriModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Modal } from 'antd';
+import QRCode from 'qrcode.react';
+import Copy from 'components/Copy';
+import { AppState } from 'store/reducers';
+
+interface StateProps {
+  node: AppState['node']['nodeInfo'];
+}
+
+interface OwnProps {
+  isOpen?: boolean;
+  onClose(): void;
+}
+
+type Props = StateProps & OwnProps;
+
+class NodeUriModal extends React.Component<Props> {
+  render() {
+    const { node, isOpen, onClose } = this.props;
+    const text =
+      node === null ? '' : !!node.uris[0] === false ? node.identity_pubkey : node.uris[0];
```

Comment:
When I mentioned a falsy check, I didn't mean literally `=== false`. You want to catch all sorts of cases, false, undefined, null, empty string etc. So in those cases, you just do `node && node.uris && node.uris[0] ? /* true case here */ : /* false case here */`. That catches any instance where node or node.uris[0] isn't what we want.

Reply:
Ahh, I see. My bad. (^_^)
```Typescript
const text = node === null ? '' :
      node && node.uris && node.uris[0] ?
      node.uris[0] : node.identity_pubkey;
const title = node === null ? '' :
      node && node.uris && node.uris[0] ?
      'Node URI': 'Pubkey';